src/main/java/com/deeme/modules/PaladiumModule.java

```diff
@@ -216,9 +218,10 @@ private void sell() {
         if (hero.map != SELL_MAP) main.setModule(new MapModule()).setTarget(SELL_MAP);
         else bases.stream().filter(b -> b.locationInfo.isLoaded()).findFirst().ifPresent(base -> {
             if (drive.movingTo().distance(base.locationInfo.now) > 200) { // Move to base
-                double angle = base.locationInfo.now.angle(hero.locationInfo.now) + Math.random() * 0.2 - 0.1;
-                drive.move(Location.of(base.locationInfo.now, angle, 100 + (100 * Math.random())));
-            } else if (!hero.locationInfo.isMoving() && oreTrade.showTrade(true,base)
+                double angle = ThreadLocalRandom.current().nextDouble(Math.PI * 2);
+                double distance = 100.0 + ThreadLocalRandom.current().nextDouble(100.0);
```

Comment:
use Math.random() or a Random object, but don't rely on ThreadLocalRandom because it really isn't needed

```suggestion
                double angle = Math.random() * Math.PI * 2;
                double distance = 100.0 + Math.random() * 100.0;
```